### ISSUE TITLE
JFFS2 support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rampageX/firmware-mod-kit#115]
```
See below log:

# /opt/firmware-mod-kit/trunk/extract-firmware.sh VMG8324-AAHA3.bin 
Firmware Mod Kit (build-ng) 0.82, (c)2011-2013 Craig Heffner, Jeremy Collake

Scanning firmware...

Scan Time:     2013-09-24 15:03:22
Signatures:    193
Target File:   /tmp/VMG8324-AAHA3.bin
MD5 Checksum:  4e651121a42ffc0f0234d5d85edba3f4

DECIMAL     HEX         DESCRIPTION
--------------------------------------------------------------------------------
-----------------------
131072      0x20000     JFFS2 filesystem, big endian 

Extracting 131072 bytes of  header image at offset 0
Extracting jffs2 file system at offset 131072
Extracting 2672 byte footer from offset 22038131
Unsupported file system 'jffs2'! Quitting...
# 

TL;DR JFFS2 support would be beautiful :)

```

Original issue reported on code.google.com by `the.info...@gmail.com` on 24 Sep 2013 at 3:05
